apps/docker/Dockerfile

```diff
@@ -1,38 +1,37 @@
-# Librelingo is not compatible with Node 16, using Node 14.18.3 image instead
-FROM node:14.18.3 
+FROM archlinux:base-20220227.0.49015
 
-# Setup Librelingo
+# Setup packages
+RUN pacman -Syu --noconfirm git nodejs-lts-fermium gcc yarn python3 python-pip
+
+# Setup Poetry and LibreLingo Python Packages
+RUN curl -sSL https://install.python-poetry.org | python3 - && \
+  cp /root/.local/bin/poetry /bin && \
+  pip install librelingo-yaml-loader && \
+  pip install librelingo-json-export
+  
+# Git clone LibreLingo repo
 RUN git clone https://github.com/LibreLingo/LibreLingo.git
```

Comment:
actually, I realize now that we should not git clone the repo in the Dockerfile, right? I mean, we should simply add the relevant files from the folder the user already has, so that local changes can also be tested using Docker?

This PR can be accepted without that fix though, because this "mistake" already existed before

Reply:
My original idea for the docker image was to test courses in Librelingo by just running one command that would setup the docker container and volume mount in, but I do agree that the docker image should be used to help work on development on librelingo. Maybe, make two docker images. One with the git clone for course testing in Librelingo and another for development

Reply:
I could perhaps setup a label that allows you to set if do or don't want the repo git cloned when the docker-entrypoint.sh runs